src/app/projects/projects-list/projects-list.component.ts

```diff
@@ -10,22 +10,27 @@ export class ProjectsListComponent implements OnInit {
 
   projects;
 
-  constructor(private projectsData: ProjectServiceService) {
+  constructor(private projectsData: ProjectServiceService) { }
 
+  userProjects = false;
+  sortList = ['all', 'rating', 'name', 'date'];
+  sort = this.sortList[0];
+  sortTypeValue = 'false';
+
+  selectSort(event) {
+    if (event === 'name') {
```

Comment:
Do we really need this check?

Reply:
we really need this becouse in select input i put string 'name' but in our DB property that property is 'projectName' so to make this string not so ugly for user i need to do check with another property.